helm/overlay/patches/patch_deployment.yaml

```diff
@@ -9,23 +9,25 @@ spec:
     spec:
       containers:
         - name: manager
+          imagePullPolicy: "{{ .Values.deployment.imagePullPolicy | default Always }}"
           env:
             - name: OTEL_COLLECTOR_URL
               value: "{{ .Values.otelCollector.url }}"
 
 
-#---
-#apiVersion: apps/v1 #example patch for certmanager
-#kind: Deployment
-#metadata:
-#  name: cert-manager
-#  namespace: system
-#spec:
-#  template:
-#    spec:
-#      containers:
-#        - name: manager
-#          env:
+---
+apiVersion: apps/v1 #example patch for certmanager
+kind: Deployment
+metadata:
+  name: cert-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          imagePullPolicy: "{{ .Values.deployment.imagePullPolicy | default Always }}"

```

Comment:
```suggestion
          imagePullPolicy: {{ .Values.deployment.imagePullPolicy | default "Always" }}

```

Reply:
Quotes around the template syntax are already used in other places of the file. Should I also update those following your suggestions? Ref: [Here](https://github.com/keptn/lifecycle-toolkit/blob/48b0dbcdb14ca08e54623c411759a23449184354/helm/overlay/patches/patch_deployment.yaml#L14)

Reply:
yes please remove them and introduce a default value there, thank you for noticing!

actually this:

`{{ .Values.otelCollector.url | default "otel-collector:4317" }}`

Thank you!

Reply:
updated